plugins/rtmp-services/data/services.json

```diff
@@ -387,88 +387,134 @@
             ]
         },
         {
-            "name": "Vaughn Live / iNSTAGIB",
+            "name": "Vaughn Live",
```

Comment:
If you change the name, you should add the old name as an entry in `alt_names` to avoid breaking user configurations.